app/src/common/shared/com/igalia/wolvic/browser/PromptDelegate.java

```diff
@@ -317,9 +318,24 @@ public void dismiss() {
     @Nullable
     @Override
     public WResult<PromptResponse> onDateTimePrompt(@NonNull WSession session, @NonNull DateTimePrompt prompt) {
-        // TODO implement date/time picker
         final WResult<PromptResponse> result = WResult.create();
-        result.cancel();
```

Comment:
@svillar Now I believe the issue is caused by `onDateTimePrompt()` not properly called. onDateTimePrompt() will only be called once and will not get called again when we click on the input again. The only way for it to be called again is to re-focus the input.

The issue already exists even without this PR (where we only have the result canceled without showing anything), so it's not about the change of focus on our side.

Reply:
Awesome thanks. Mind opening a new bug with this info? (even just referring to this bug/comment)

Reply:
Issue opened at https://github.com/Igalia/wolvic/issues/896